jni/instruments/sampledinstrument.cpp

```diff
@@ -58,21 +58,6 @@ SampledInstrument::~SampledInstrument()
 
 /* public methods */
 
-bool SampledInstrument::removeEvent( BaseAudioEvent* audioEvent, bool isLiveEvent )
-{
-    bool removed = false;
-
-    if ( audioEvent != 0 )
-    {
-        removed = BaseInstrument::removeEvent( audioEvent, isLiveEvent );
-#ifndef USE_JNI
-        delete audioEvent;
-        audioEvent = 0;
-#endif
-    }
-    return removed;
-}
```

Comment:
No need to override this, we want this behaviour to be managed by the BaseInstrument